src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -0,0 +1,219 @@
+  private WebElement tabber;
+  @FindBy(css = ".tabbertab .image")
+  private WebElement tabberImage;
+  @FindBy(css = ".portable-infobox-item-value ul li")
+  private List <WebElement> unorderedElementList;
+  @FindBy(css = ".portable-infobox-item-value ol li")
+  private List <WebElement> orderedElementList;
+  @FindBy(css = ".reference")
+  private WebElement referenceElements;
+  @FindBy(css = ".portable-infobox-item-label")
+  private WebElement h3Elements;
+  @FindBy(css = "#CreatePageModalDialog")
+  private WebElement createArticleModal;
+  @FindBy(css = "button[data-event=create]")
+  private WebElement addAPageButton;
+  @FindBy(css = ".portable-infobox-item-value .newcategory")
+  private WebElement categoryLink;
+  @FindBy(css = ".portable-infobox-item-label")
+  private WebElement itemLabel;
+  @FindBy(css = ".portable-infobox-item-value")
+  private WebElement itemValue;
+  @FindBy(css = "h3.portable-infobox-item-label.portable-infobox-secondary-font")
+  private WebElement horizontalItemLabel;
+  @FindBy(css = "div.portable-infobox-item-value")
+  private WebElement horizontalItemValue;
+
+
+  public PortableInfoboxPageObject(WebDriver driver) {
+    super(driver);
+    PageFactory.initElements(driver, this);
+  }
+
+  public String getBackgroundColor() {
+    return infoboxLayout.getCssValue("background-color");
+  }
+
+  public List<WebElement> getBoldElements(){
+    return boldElements;
+  }
+
+  public List<WebElement> getItalicElements() {
+    return italicElements;
+  }
+
+  public List<WebElement> getHeaderElements() {
+    return pInfoTitleH3;
+  }
+
+  public void verifyImagePresence() {
+    waitForElementByElement(pInfoImage);
+    Assertion.assertEquals(checkIfElementOnPage(pInfoImage), true);
+  }
+
+  public void verifyTabberPresence() {
+    waitForElementByElement(tabber);
+    Assertion.assertEquals(checkIfElementOnPage(tabber), true);
+  }
+
+  public void verifyTabberImagePresence() {
+    waitForElementByElement(tabberImage);
+    Assertion.assertEquals(checkIfElementOnPage(tabberImage), true);
+  }
+
+  public void verifyInfoboxTitlePresence() {
+    waitForElementByElement(pInfoTitle);
+    Assertion.assertEquals(checkIfElementOnPage(pInfoTitle), true);
+  }
+
+  public void verifyLightboxPresence() {
+    waitForElementByElement(lightbox);
+    Assertion.assertEquals(checkIfElementOnPage(lightbox), true);
+  }
+
+  public String getExternalLinkRedirectTitle() {
+    waitForElementByElement(pInfoExternalLink);
+    return pInfoExternalLink.getAttribute("href");
+  }
+
+  public String getInternalLinkRedirectTitle(int index) {
+    waitForElementByElement(pInfoInternalLinks.get(index));
+    return pInfoInternalLinks.get(index).getAttribute("href");
+  }
+
+  public WebElement getItemLabel() {
+    return itemLabel;
+  }
+
+  public WebElement getItemValue() {
+    return itemValue;
+  }
+
+  public WebElement getOrderedListItem(int index) {
+    WebElement orderedListItem = orderedElementList.get(index);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "orderedListItem". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)